File: Python/Menor valor/desafio.py
# Crie um programa que peça ao usuário três números e retorne o menor número entre eles


while True:
    try:
        n1 = int(input('[+] Digite o 1° valor: '))
        n2 = int(input('[+] Digite o 2° valor: '))
        n3 = int(input('[+] Digite o 3° valor: '))

        if (n1 < n2) and (n1 < n3):
            print(f'[+] O menor valor digitado foi o {n1}')
            break 
        
    except ValueError:
        print('[+] Digite um número inteiro valido! Tente novamente.')
        continue

File: JavaScript/Maior-valor/desafio.js
// Crie uma função chamada maiorNumero que recebe três números como parâmetros e retorna o maior entre eles.

function maiorNumero(n1, n2, n3) {
    if (n1 >= n2 && n1 >= n3) {
        console.log(`O número maior é o ${n1}`)
    
    } else if (n2 >= n1 && n2 >= n3) {
        console.log(`O número maior é o ${n2}`)
    
    } else {
        console.log(`O número maior é o ${n3}`)
    
    }
}

maiorNumero(1, 2, 3) 
